api/client/lib.py

```diff
@@ -296,11 +296,11 @@ def list_available(access_token, api_host, selected_entities):
         raise Exception(resp.text)
 
 
-@memoize(maxsize=None)
-def lookup(access_token, api_host, entity_type, entity_id):
-    url = '/'.join(['https:', '', api_host, 'v2', entity_type, str(entity_id)])
+def lookup(access_token, api_host, entity_type, entity_ids):
+    url = '/'.join(['https:', '', api_host, 'v2', entity_type])
```

Comment:
i'd suggest backward compatibility here too for older clients as @itsjohnward brought up in stand up..

Reply:
I just went ahead and moved the translation bit into here since if we *do* want lib to always be backwards compatible, then it's necessary, but if we decide it *doesn't* need to be then we can always just refactor it back later. So, erred on the safe side